twitter-export-image-fill.py

```diff
@@ -122,6 +141,11 @@
       if (not args.include_retweets) and retweeted:
         continue
 
+      # Download avatar for tweet and retweet
```

Comment:
I like that you added it, but I don’t think we need to broadcast avatar counts when processing – I think it is a bit TMI and makes this too complicated.

However, we should talk about this in the FAQ/README!